{
  "0 debug pnpm:scope": {
    "selected": 1
  },
  "1 info pnpm": {
    "err": {
      "name": "Error",
      "message": "Command failed with ENOENT: cache verify\nspawn cache ENOENT",
      "code": "ENOENT",
      "stack": "Error: Command failed with ENOENT: cache verify\nspawn cache ENOENT\n    at ChildProcess._handle.onexit (node:internal/child_process:286:19)\n    at onErrorNT (node:internal/child_process:484:16)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"
    }
  },
  "2 error pnpm": {
    "errno": -2,
    "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
    "syscall": "spawn cache",
    "path": "cache",
    "spawnargs": [
      "verify"
    ],
    "originalMessage": "spawn cache ENOENT",
    "shortMessage": "Command failed with ENOENT: cache verify\nspawn cache ENOENT",
    "command": "cache verify",
    "escapedCommand": "cache verify",
    "failed": true,
    "timedOut": false,
    "isCanceled": false,
    "killed": false,
    "prefix": "/Users/ngmm/m-workspace/admin-v2",
    "err": {
      "name": "Error",
      "message": "Command failed with ENOENT: cache verify\nspawn cache ENOENT",
      "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
      "stack": "Error: Command failed with ENOENT: cache verify\nspawn cache ENOENT\n    at ChildProcess._handle.onexit (node:internal/child_process:286:19)\n    at onErrorNT (node:internal/child_process:484:16)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"
    }
  }
}